src/NodaTime/CalendarSystem.cs

```diff
@@ -634,6 +634,21 @@ public int GetMonthsInYear(int year)
             return YearMonthDayCalculator.GetMonthsInYear(year);
         }
 
+        /// <summary>
+        /// Returns the number of weeks in the given week-year.
+        /// </summary>
+        /// <param name="weekYear">The week-year to determine the number of weeks for.</param>
+        /// <exception cref="ArgumentOutOfRangeException"><paramref name="weekYear"/> is outside
+        /// the range of this calendar. </exception>
+        /// <returns>The number of weeks in the given week-year.</returns>
+        public int GetWeeksInWeekYear(int weekYear)
+        {
+            // TODO: Is this really valid? It means we can't necessarily round-trip. Maybe be more lenient, but then check the resulting days-since-epoch.
```

Comment:
I'm not sure I follow the reason we can't round-trip?

Perhaps add a test that demonstrates the limitation we have here?